styles.css

```diff
@@ -0,0 +1,224 @@
+*{
+	box-sizing: border-box;
+}
+
+html{
+	font-size: 20px;
+}
+
+.container {
+	width: 1600px;
+	margin: 0 auto;
+}
+
+
+/* navbar section*/
+nav{
+	display: flex;
+	background: #E5E2DE;
+	height: 60px;
+	margin: 10px 0;
+	justify-content: center;
+	align-items: center;
+	
+}
+
+nav > div{
+	display: flex;
+}
+
+.navigation{
+	padding: 5px;
+	margin: 5px;
+		
+}
+
+.navBorder{
+	border: 2px solid #777674;
+}
+
+.navigation > a{
+	text-decoration: none;
+}
+
+.navigation > a:visited{
+	color: #000000;
+}
+
+.navigation > a:link{
+	color: #000000;
+}
+
+/* title section*/
+header{
+	display: flex;
+	height: 700px;
+	justify-content: center;
+	align-items: center;
+	padding: 180px 0;
+	margin: 0 25px;
+	background-image: url("titleBackground2.jpg");
+	background-size: cover;
+}
+
+header > div{
+	width: 750px;
+	color: #FEE6D3;
+}
+
+.titleBackgroundFooter{
+	display: flex;
+	justify-content: center;
+}
+
+h1{
+	font-size: 3.0rem;
+}
+
+/* About me section*/
+.aboutMe{
+	margin: 100px 100px;
+}
+
+h2{
+	margin: 0;
+	font-size: 2.0rem;
+}
+
+
+/*Projects section*/
+main{
+	width: 95%;
+	margin: 0 auto;
+}
+
+.projectsSection > h2 {
+	text-align: center;
+	padding: 20px;
+}
+
+.projectsSection > div{
+	display: flex;
+	flex-flow: wrap;
+	justify-content: space-evenly;
+}
+
+.project{
+	width: 240px;
+	margin: 30px 20px;
+}
+
+.projectImage{
+	height: 240px;
+	width: 240px;
+	display: flex;
+	justify-content: center;
```

Comment:
doesnt seem to be doing anything, remove

Reply:
I think this only does something for images that are taller than they are wide.  Since I don't have such images it doesn't do anything right now but I think it'll be useful to keep in case I do decide to put in some tall images.

Reply:
sounds good